packages/rabbitmq/test/jest-setup/docker.ts

```diff
@@ -0,0 +1,50 @@
+import { connect } from 'amqplib';
+import { imageExists, pullImageAsync } from 'dockerode-utils';
+import Docker from 'dockerode';
+
+const CONTAINER_IMAGE = 'rabbitmq:3.8-alpine';
+const CONTAINER_NAME = 'cardano-rabbitmq-test';
+
+export const removeContainer = async () => {
+  const docker = new Docker();
+  try {
+    const container = await docker.getContainer(CONTAINER_NAME);
+    try {
+      await container.stop();
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    } catch (error: any) {
+      // 304 = container already stopped
+      if (error.statusCode !== 304) throw error;
+    }
+    await container.remove({ v: true });
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  } catch (error: any) {
+    // 404 = container not found
+    if (error.statusCode !== 404) throw error;
+  }
+};
+
+export const setupContainer = async () => {
+  const docker = new Docker();
+  const needsToPull = !(await imageExists(docker, CONTAINER_IMAGE));
+  if (needsToPull) await pullImageAsync(docker, CONTAINER_IMAGE);
+  await removeContainer();
+  const container = await docker.createContainer({
+    HostConfig: { PortBindings: { '5672/tcp': [{ HostPort: '5672/tcp' }] } },
```

Comment:
For our future refactor, it would be good to get a random available host port to reduce chance of a clash.

Reply:
Crystal clear! I'll handle this in the next days.

Reply:
As agreed on slak, I resolve this